handsontable/src/helpers/object.js

```diff
@@ -10,7 +10,8 @@ export function duckSchema(object) {
   let schema;
 
   if (Array.isArray(object)) {
-    schema = [];
+    schema = object.length ? new Array(object.length).fill(null) : [];
```

Comment:
Can I ask to cover that change with a unit test ([object.unit.js](https://github.com/handsontable/handsontable/blob/feature/issue-1/handsontable/src/helpers/__tests__/object.unit.js#L69))?